docs/views/layout.html

```diff
@@ -40,26 +40,48 @@
 {% block header %}
   {% include "includes/cookie-banner.html" %}
   {# Set serviceName in config.js. #}
-  {{ govukHeader({
-    "homepageUrl": "/",
-    "serviceName": serviceName,
-    "serviceUrl": "/",
-    "containerClasses": "govuk-width-container",
-    "navigation": [
-      {
-        "href": "/docs/install",
-        "text": "Install"
-      },
-      {
-        "href": "/docs/tutorials-and-examples",
-        "text": "Tutorials and examples"
-      },
-      {
-        "href": "/docs/about",
-        "text": "About"
-      }
-    ]
-  }) }}
+{# sadly cannot use the macro to pass data- attributes to items #}
```

Comment:
Can we link to the issue please? Maybe label this ina  todo format

"TODO: Use the macro when data attributes can be passed to individual items: https://..."